<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cbAutoUpdate">
       <property name="text">
        <string>Check For Updates Automatically</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbUpdate">
       <property name="text">
        <string>Check Now</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbGizmos">
     <property name="text">
      <string>Show Transform Gizmos</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl2">
     <item>
      <widget class="QLabel" name="lbImgExt">
       <property name="text">
        <string>Default Image Filetype:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cobImgExt"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbAscImport">
     <property name="title">
      <string>ASC Import</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="cbAscPrefixes">
        <property name="text">
         <string>Remove 'ZM_' And 'Bip01' Prefixes</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbXmacImport">
     <property name="title">
      <string>_XMAC Import</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cbXmacCollisionMesh">
        <property name="text">
         <string>Remove Collision Mesh</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl1">
     <item>
      <widget class="QPushButton" name="pbRestoreDefaults">
       <property name="text">
        <string>   Restore Default   </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hs1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
